### DISCUSSION TITLE
The financialEvents interface does not return nextToken if it exceeds 100 pages

### DISCUSSION THREAD

**Discussion by wzc2012** [amzn/selling-partner-api-models#3860] [General]
When I call the {listFinancialEventsByGroupId} interface, if the results returned by GroupId exceed 100 pages, the interface will not return NextToken, resulting in the inability to obtain data after 100 pages.

**Comment by marvs** [comment]
Same behavior here but in the listFinancialEvents endpoint. The NextToken parameter is not returned beyond 100 pages, so tallying the financial data results in a much less amount than the expected amount.

**Comment by rabbiveesh** [comment]
this has catastrophic consequences for any seller who's relying on this data + does a larger amount of sales.
